@AH-64D Apache Longbow/Addons/fza_ah64_sfmplus/functions/aero/fn_aeroStabilator.sqf

```diff
@@ -28,21 +28,21 @@ private _sfmPlusConfig = _cfg >> "Fza_SfmPlus";
 
 if (!local _heli) exitWith {};
 
-private _deltaTime      = fza_ah64_fixedTimeStep;
-private _heliCOM        = getCenterOfMass _heli;
-private _numElements    = 5;
-private _liftCurveSlope = 5.7;
-private _baseDragCoef   = 0.025;
-private _K              = 0.07;
-private _stabPos        = [0.0, -6.45, -1.85];//_heli getVariable "fza_sfmplus_stabPos";
-private _span           = _heli getVariable "fza_sfmplus_stabWidth";
-private _chord          = _heli getVariable "fza_sfmplus_stabLength";
-private _chordLinePos   = 0.25;
+private _deltaTime       = fza_ah64_fixedTimeStep;
+private _heliCOM         = getCenterOfMass _heli;
+private _numElements     = 5;
+private _airfoilTable    = getArray (_sfmPlusConfig >> "stabAirfoilTable");
+private _stabPos         = [0.0, -6.45, -1.85];//_heli getVariable "fza_sfmplus_stabPos";
+private _span            = _heli getVariable "fza_sfmplus_stabWidth";
+private _chord           = _heli getVariable "fza_sfmplus_stabLength";
+private _chordLinePos    = 0.25;
 
 private _stabOutputTable = [[]];
-private _theta           = 0.0;
+private _desiredTheta    = 0.0;
+private _theta           = _heli getvariable "fza_ah64_stabilatorPosition";
```

Comment:
This is a duplication of line 85, this should either be the default 0 or replace lines 84-90 with
```sqf
if (_stabDamage < SYS_STAB_DMG_THRESH && _dcBusOn) then {
    _desiredTheta = [_stabOutputTable, (_heli getVariable "fza_sfmplus_vel2D") * KNOTS_TO_MPS] call fza_fnc_linearInterp select 1;
    _theta = [_theta, _desiredTheta, (1.0 / 1.5) * _deltaTime] call BIS_fnc_lerp;
    _heli setVariable ["fza_ah64_stabilatorPosition", _theta];
};
```